packaging/src/docker/conf/hive-site.xml

```diff
@@ -60,4 +60,20 @@
         <name>metastore.metastore.event.db.notification.api.auth</name>
         <value>false</value>
     </property>
+    <property>
+        <name>iceberg.catalog</name>
+        <value>rest</value>
+    </property>
+    <property>
+        <name>iceberg.catalog.rest.type</name>
```

Comment:
`iceberg.catalog.type`

Reply:
At iceberg.catalog the value is "rest", so here it must be iceberg.catalog.rest.type.